test/modules/noakes/selfReference2.chpl

```diff
@@ -28,6 +28,7 @@ module selfReference {
 
 module internal {
   proc test0() {
+    use internal only;
```

Comment:
Here was the other instance of self-reference that I noticed

Reply:
Thanks, I've removed it now and added a TODO